grails-app/controllers/org/pih/warehouse/inventory/InventoryItemController.groovy

```diff
@@ -739,6 +739,7 @@ class InventoryItemController {
         log.info "Params " + params
         def itemInstance = InventoryItem.get(params.id)
         def productInstance = Product.get(params?.product?.id)
+        def date = new Date("01/01/2000")
```

Comment:
Ok I see this was probably a mistake since it's not used. But then I see that you sprinkled similar declarations throughout the code. I don't love this, but haven't had a chance to think through how it should be done. This at least needs to be a configurable property (Config.groovy). But I don't even really love that. 
```
openboxes.expirationDate.minValue = new Date("01/01/2000") 
```
I think the validation needs to happen in the domain constraints (i.e. inventory item and any other domain that contains an expiration date). And we could probably want to create a global constraint. 
```
grails.gorm.default.constraints = {
   expirationDateConstraint(nullable:true, min: grailsApplication.config.openboxes.expirationDate.minValue)
}
```
InventoryItem.groovy 
```
static constraints = {
    expirationDate(shared:expirationDateConstraint)
}
```

See https://grails.github.io/grails2-doc/1.3.9/ref/Constraints/Usage.html

@awalkowiak Do you have any thoughts?

Reply:
Moving this to config seems good, but I don't understand the idea of having minimum expiration date of 01/01/2000 and max 20 years from current year.